docs/usage.md

```diff
@@ -0,0 +1,42 @@
+# Usage
+
+The Data Registry JupyterLab extension adds two new UI elements, a Data Explorer in the left pane, and a Data Browser in the right pane:
+
+![Data registry and data browser panes](./img/both.png)
+
+The Data Registry lets you browse datasets that are available to you. It is a nested view that shows a number of actions you can take for each dataset along with any children datasets.
+
+![Show children datasets of a folder and actions](./img/registry)
+
+We currently build in support for a number of limited data types. These are meant to show examples of different types of interactions and stress test the system. They can be extended by creating custom JupyterLab extensions by third parties to add any new file types or viewsers:
```

Comment:
We currently provide built-in support for a limited number of data types. These are meant to show examples of different types of interactions and stress test the system. They can be extended by creating custom JupyterLab extensions which add new file types and/or viewers: